packages/styling/src/styles/theme.ts

```diff
@@ -97,15 +97,17 @@ function _loadFonts(theme: ITheme): { [name: string]: string } {
   const lines = {};
 
   for (const fontName of Object.keys(theme.fonts)) {
-    const font = theme.fonts[fontName];
+    // tslint:disable-next-line:no-any
+    const font = (theme.fonts as any)[fontName];
```

Comment:
Really this should be:
```suggestion
    const font = theme.fonts[fontName as keyof IFontStyles];
```

Reply:
fixed.